numpy/lib/function_base.py

```diff
@@ -3717,6 +3717,8 @@ def _ureduce(a, func, **kwargs):
     """
     a = np.asanyarray(a)
     axis = kwargs.get('axis', None)
+    out = kwargs.get('out', None)
+
```

Comment:
Total nitpick, but you might want to do here
```
if keepdims is np._NoValue:
    keepdims = False
```
so that below you don't have to worry about it (and is more readable).

Reply:
I totally agree. I did it that way since I saw that pattern `np.nanmedian` and I figured I might as well keep things consistent, but I would be happier writing it the was you suggested.

As an aside, do you know of a mechanism or function that resolves keywords with `np._NoValue` into their default value? Seems like that would be the "proper" way of handling this if it exists.

Reply:
I don't think there is such a mechanism (and it would probably hard to make it fast enough).

Reply:
Ok, well in that case I am going to update this PR with your suggestions because I think it is definitely more clear that way.